server/app.js

```diff
@@ -16,7 +15,6 @@ app.use(express.urlencoded({ extended: true }));
 app.use(cookieParser());
 
 app.use("/api/users", userRoutes);
-app.use("/api/workspaces", workspaceRoutes);
```

Comment:
For some reason, this PR removes workspace routes. Can we re-add it back?

Reply:
@danielskjo I think that's because I didn't pull from Main before requesting this PR.  I went ahead and pulled from Main and merged into this branch just now, and now I can see the code for the workspaces.